consensus/istanbul/validator/default.go

```diff
@@ -247,6 +247,27 @@ func (valSet *defaultSet) Copy() istanbul.ValidatorSet {
 	return newValSet
 }
 
+// We include the optimization described at https://arxiv.org/pdf/1901.07160.pdf as “PM-6” and
+// discussed in Lemma 22. For values of N=3F for integer F=1,2,3,.. we can tolerate a quorum
```

Comment:
Should this be `N>=3F`?
```suggestion
// discussed in Lemma 22. For values of N>=3F for integer F=1,2,3,.. we can tolerate a quorum
```

Reply:
I think no, just specifically those `N` that are exact multiples of `3F` ?

Reply:
Ah I see now. You are discussing this specific case in this paragraph.